### PR TITLE
GMP Documentation and alert: Elasticsearch

### PR DESCRIPTION
**Pull request by jsirianni**
This PR adds documentation configuration for the Elasticsearch GMP integration.

-  Added documentation.yaml
-  Added prometheus_metadata.yaml

The documentation yaml includes an alert ported from https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/tree/master/alerts/elasticsearch-gke

### CODE REVIEW

File: integrations/elasticsearch/prometheus_metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+platforms:
+  - type: GKE
+    launch_stage: GA
+    exporter_metadata:
```
Comment (EvanSimpson) [inline integrations/elasticsearch/prometheus_metadata.yaml:4]: Can you add a characteristic metric detection as well?

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

**Review by yqlu — APPROVED** [review]
👍  in terms of validating the config on a cluster myself

Tagging in @mtabasko to review the docs format. This is probably significant in the sense that it is our first "standalone" type exporter doc, so we want to ensure the template works.

File: integrations/elasticsearch/documentation.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  spec:
+    endpoints:
+    - port: prometheus
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: elasticsearch-exporter
+additional_prereq_info: |
```
Comment (mtabasko) [inline integrations/elasticsearch/documentation.yaml:54]: This seems like "additional_install_info" (it makes sense after you've seen the install config).

**Comment by mtabasko** [comment]
Other than the one small change, from "additional_prereq_info" to "additional_install_info", I don't see any problems. (We had actually done a version of Elasticsearch as a "standalone" exporter when building the template originally -- the biggest diff between "standalone" and "sidecar" in terms of the template is whether we use "config_mods" or "standalone_config" for the config info, and that all looks good here to me! Thanks.

**Review by mtabasko — APPROVED** [review]
docs look good. Thanks again!

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**